docs/rfcs/2024-02-28-pq-integration.md

```diff
@@ -0,0 +1,60 @@
+# PQ integration in chat
+
+## Problem
+
+- Group size not known when joining
+- Communicate intent and current state of each conversation
+
+## Solution
+
+### Group size not known when joining
+
+- Add to XGrpInv GroupInvitation
+  - pros: easy
+  - cons: size can change before joining, but can ignore as it's still a good estimate
+
+or
+
+- Send before introductions
+  - new protocol message
+    - XGrpIntro :: GrpIntro -> ChatMsgEvent 'Json -- (GrpIntro is a box type with Int, for possible extension)
+  - or put into XGrpInfo
+    - XGrpInfo :: GroupProfile -> GroupStats -> ChatMsgEvent 'Json -- GroupData?
+    - can update profile between invitation if it happened before joining
+    - can later add logic to "verify" stats?
+    - may be over-complicated until since there "supposed" use cases are out-of-scope / not planned / not known
+
+- What should be default if it's not known? (e.g. admin has older version)
```

Comment:
off is ok, as old version means no PQ support. Also, adding to invitations seems ok, as most groups are either small or large.